xarray/tests/test_formatting_html.py

```diff
@@ -320,6 +320,25 @@ def test_two_children(
         )
 
 
+class TestDataTreeInheritance:
+    def test_inherited_section_present(self) -> None:
+        dt = xr.DataTree.from_dict(
+            {
+                "/": None,
+                "a": None,
+            }
+        )
+        with xr.set_options(display_style="html"):
+            html = dt._repr_html_().strip()
+        # checks that the section appears somewhere
+        assert "Inherited coordinates" in html
+
+        # TODO how can we assert that the Inherited coordinates section does not appear in the child group?
+        # with xr.set_options(display_style="html"):
+        #     child_html = dt["a"]._repr_html_().strip()
+        # assert "Inherited coordinates" not in child_html
```

Comment:
I'm not really sure how to test this any better. Both the correct and incorrect behaviour would give `"Inherited coordinates"` present somewhere in the long html repr text output.